app/src/main/java/org/fossasia/openevent/general/event/EventDetailsViewModel.kt

```diff
@@ -26,6 +32,14 @@ class EventDetailsViewModel(private val eventService: EventService, private val
     val error: LiveData<String> = mutableError
     private val mutableEvent = MutableLiveData<Event>()
     val event: LiveData<Event> = mutableEvent
+    val eventSpeakers: MutableLiveData<List<Speaker>> = MutableLiveData()
+
+    fun loadEventSpeakers(id: Long) {
+        speakerService.fetchSpeakersForEvent(id)
```

Comment:
Will have no effect since you haven't subscribed

Reply:
Same

Reply:
if i subscribe to it the result of the subscription will be of no use.Will that be okay ?

Reply:
something like this
        speakerService.fetchSpeakersForEvent(id)
            .subscribeOn(Schedulers.io())
            .observeOn(AndroidSchedulers.mainThread())

Reply:
Yes, it'll be OK

Reply:
updated

Reply:
Still not subscribed